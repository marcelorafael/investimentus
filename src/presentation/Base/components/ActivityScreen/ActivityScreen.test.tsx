it('should match snapshot', () => {
  console.log('teste')
})
