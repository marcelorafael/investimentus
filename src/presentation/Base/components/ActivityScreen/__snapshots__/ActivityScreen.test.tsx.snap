// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base/ActivityScreen should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFF",
        },
      ]
    }
  >
    
  </Text>
  <View
    style={
      Object {
        "marginTop": 10,
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#FFF",
            "fontSize": 50,
          },
          undefined,
          Object {
            "fontFamily": "FontAwesome",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï…š
    </Text>
  </View>
</View>
`
